sandbox/modules/identity/src/main/java/org/opensearch/identity/IdentityPlugin.java

```diff
@@ -131,6 +131,9 @@ public List<Setting<?>> getSettings() {
         settings.add(
             Setting.simpleString(ConfigConstants.IDENTITY_CONFIG_INDEX_NAME, Setting.Property.NodeScope, Setting.Property.Filtered)
         );
+        settings.add(
+            Setting.simpleString(ConfigConstants.IDENTITY_ADMIN_USERNAME, "admin", Setting.Property.NodeScope, Setting.Property.Filtered)
```

Comment:
Why add a setting, shouldn't we always created an `admin` account by default?  Its what we do in the security plugin today.

Reply:
An additional layer of security. By having the admin username be configurable, it may not be assumed universally that `admin` is the admin username.